coreneuron/gpu/nrn_acc_manager.cpp

```diff
@@ -872,136 +774,6 @@ void update_nrnthreads_on_host(NrnThread* threads, int nthreads) {
 #endif
 }
 
-void update_nrnthreads_on_device(NrnThread* threads, int nthreads) {
```

Comment:
I think this one is really not used at the moment. I am OK with removing it, we can recover it from git history if needed...if it is not used/tested, chanced are this implementation has bugs anyway.